src/main/java/org/broadinstitute/hellbender/cmdline/argumentcollections/IntervalArgumentCollection.java

```diff
@@ -55,7 +55,10 @@
      * (e.g. -XL myFile.intervals).
      * @return strings gathered from the command line -XL argument to be parsed into intervals to exclude
      */
-    @Argument(fullName = EXCLUDE_INTERVALS_LONG_NAME, shortName = "XL", doc = "One or more genomic intervals to exclude from processing", optional = true, common = true)
+    // Interval list files such as Picard interval lists are structured and require specialized parsing that
+    // is handled by IntervalUtils, so use suppressFileExpansion to bypass command line parser auto-expansion.
+    @Argument(fullName = EXCLUDE_INTERVALS_LONG_NAME, shortName = "XL", doc = "One or more genomic intervals to exclude from processing",
```

Comment:
Can you add a named constant for `XL`, while you're here?

Reply:
Done.